tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1748,6 +1808,7 @@ def grayscale_to_rgb(images, name=None):
 
   Outputs a tensor of the same `DType` and rank as `images`.  The size of the
   last dimension of the output is 3, containing the RGB value of the pixels.
+  The input images' last dimension must be size 1.
```

Comment:
I can't tell when reading this code whether this is a new requirement or not.

If it's not, then we cannot make this change as it'll break backward compatibility.

Reply:
This is not a new requirement (see [the current version](https://github.com/tensorflow/tensorflow/blob/r1.13/tensorflow/python/ops/image_ops_impl.py#L1543)). Currently, the function crashes whenever the last dimension does not have size 1 (see [issue 26324](https://github.com/tensorflow/tensorflow/issues/26324))